### DISCUSSION TITLE
No module named 'bertopic'

### DISCUSSION THREAD

**Discussion by rizkiamandaputri** [MaartenGr/BERTopic#1630] [Q&A]
![Command Prompt 15_11_2023 09_07_26](https://github.com/MaartenGr/BERTopic/assets/66537975/662a3d1f-c37a-42bb-9f86-ec26a737060d)

How can i fix it?

**Comment by MaartenGr** [comment]
If it cannot find the module, then it is not installed. Starting from a completely fresh environment and installing BERTopic there generally helps. Moreover, make sure to select the right environment when running your code.
